packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -0,0 +1,112 @@
+
+#### Timeline graph
+
+The timeline graph shows the usage over the last week. When a rate violation occurs, you can select a day from the past week in the drop-down menu on the timeline graph and compare the same point in time with a day from the past week.
+
+#### Bar graph
+
+The **Top Requests by IP** bar graph shows the top 10 IP addresses that are consuming the most traffic for the endpoint. This view enables you to check if there is a single IP or multiple IPs that are creating the spike in traffic.
+
+Click once on an IP address bar to view the traffic for that IP address on the line graph. Double-click on any IP address bar to view the traffic for all the IP addresses on the line graph.
+
+## Investigate the rate limit violation
+
+Each graph, either by itself or combined, provides you with multiple views that display the information about the rate limit violation. You can also refer to the help information on the dashboard when you are trying to figure out the rate limit violation to see if it fits into one of the common scenarios most likely to happen.
+
+### Combination of the line graph and bar graph view
+
+The line graph shows usage in the past hour and allows you to focus on a particular point during the hour when an event occurred and caused a spike in the rate limit. The bar graph identifies the top 10 IP addresses that are consuming the most traffic. From there, you can obtain more data, such as observing if there is a sudden spike in usage of one IP while all the other IPs remain relatively normal in usage. You can then identify the offending IP responsible for the spike in traffic for which you should take appropriate action. Alternatively, you may determine that the rate limit may just have a higher rate limit usage and no offending issue, and a higher rate limit is required for the endpoint.
+
+The line graph can show the number of unique IPs that are creating traffic at a given minute. For example, the line graph can show many users are signing in at the same time, perhaps to attend a company-wide meeting. By comparing the number of unique IPs to the number of unique requests, you can see almost the same number of IPs as the number of requests. This means that there are a lot of IPs creating the traffic and not just one IP consuming all the traffic.
+
+### Combination of the line graph and timeline graph view
+
+The timeline graph gives you an overview of the usage from the days in the past week so that you can drill into other time periods from the previous week. You can select one day at a time within the past week from the drop-down menu and view one hour at a time in the line graph. These views enable you to understand whether this rate limit was an abnormal spike in usage compared to fairly regular usage up to that point in time.
```

Comment:
```suggestion
The timeline graph gives you an overview of the usage from the days in the past week so that you can drill in to other time periods from the previous week. You can select one day at a time within the past week from the drop-down menu and view usage one hour at a time in the line graph. These views enable you to understand whether this rate limit was an abnormal spike in usage compared to fairly regular usage up to that point in time.
```